client/src/components/Modal/tests/Modal-test.js

```diff
@@ -0,0 +1,89 @@
+/* global jest, test, describe, it, expect */
+
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react';
+import Modal from '../Modal';
+
+let props;
+
+/** Component that printouts all of its props */
+const PrintProps = (componentName) => ({ children, ...propsToPrint }) => (
+  <div>
+    <div>{componentName}</div>
+    <ul>
+      {Object
+        .entries(propsToPrint)
+        .map(([key, value]) => (<li key={key}>{key}: {value.toString()}</li>))
+      }
+    </ul>
+    {children}
+  </div>
+);
+
+beforeEach(() => {
+  props = {
+    isOpen: true,
+    className: 'my-modal',
+    modalClassName: 'my-modal-dialog',
+    size: 'sm',
+    onClosed: jest.fn(),
+    title: 'Hello World!',
+    showCloseButton: true,
+  };
+});
+
+test('Modal renders', () => {
+  const root = render(
+    <Modal {...props}>My Content</Modal>
+  );
+
+  const modal = root.getByRole('dialog');
+  expect(modal.style.getPropertyValue('display')).toBe('block');
+  expect(modal.classList).toContain('my-modal-dialog');
+  expect(modal.classList).toContain('modal');
+
+  const document = root.getByRole('document');
+  expect(document.classList).toContain('modal-sm');
+
+  const header = root.getByRole('heading');
+  expect(header.textContent).toContain('Hello World!');
+
+  const body = root.getByText('My Content');
+  expect(body).toBeTruthy();
```

Comment:
```suggestion
  const renameThis = root.getByRole('document');
  expect(renameThis.classList).toContain('modal-sm');

  const header = root.getByRole('heading');
  expect(header.textContent).toContain('Hello World!');

  const content = root.getByText('My Content');
  expect(content).not.toBeNull();
```
Can you not use the words 'document' or 'body' for variable names here, it's really confusing as they correspond to other things in the regular DOM

Also instead of `.toBeTruthy()` use `.not.ToBeNull()` to match other tests in the codebase